hs_file_types/models/raster.py

```diff
@@ -401,19 +508,33 @@ def raster_file_validation(raster_file):
         error_info.append("Invalid file mime type found.")
 
     if not error_info:
-        files_ext = [os.path.splitext(path)[1] for path in new_resource_files_to_add]
+        if ext == ".zip":
+            # in case of zip, there needs to be more than one file extracted out of the zip file
+            if len(new_resource_files_to_add) < 2:
+                error_info.append("Invalid zip file. Seems to contain only one file. "
+                                  "Multiple tif files are expected.")
+                return error_info, []
+
+        files_ext = [os.path.splitext(path)[1].lower() for path in new_resource_files_to_add]
+        if files_ext.count('.vrt') > 1:
+            error_info.append("Invalid zip file. Seems to contain multiple vrt files.")
+            return error_info, []
 
         if set(files_ext) == {'.vrt', '.tif'} and files_ext.count('.vrt') == 1:
```

Comment:
I think we're also supposed to support .tiff

Reply:
I am not sure why '.tiff' was not supported as part of the Raster resource type which is one of the earliest resource types implemented in hydroshare.

Reply:
I have this updated in the auto-aggregate work.